.jenkins/actions/run_tests.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+envloc=$1
+RUN_CMD_FILE=$2
+maxsleep=9000
+if [ "`which sbatch`" != "" ] ; then
```

Comment:
should we not check for the hostname? This works for now but the other options seems a bit more sustainable for the future

Reply:
Oli was suggesting we try to not add case switches on hostnames in all the directories, but rather rely on the buildenv repo to get machine specific settings. This is a bit different though, because we want to submit a slurb job when we have slurm, and just run it when we don't which sbatch might not be the best here, I agree, but am not sure what else to put there.

Reply:
I was thinking this file could be something we pull out of this repo and put into env -- functionality to run a slurm job, or run the code directly if it's not there

Reply:
instead of 'which sbatch' though -- we could add to buildenv a variable 'use_slurm' that we could trigger off instead? then daint would be true, gce-cpu would be false. I would just need @ofuhrer to give me permissions to push to buildenv and open a pr there

Reply:
I have altered this, and pushed a lot of the slurm functionality into buildenv: https://github.com/VulcanClimateModeling/buildenv/pull/1, demonstrated here: https://jenkins.ginko.ch/job/gt4py-develop/71/

Reply:
I am open to changing this differently, let me know what you think!